src/sentry/integrations/fake_log_integration/metric_alerting/__init__.py

```diff
@@ -0,0 +1,49 @@
+from sentry.constants import ObjectStatus
+from sentry.incidents.action_handlers import DefaultActionHandler
+from sentry.incidents.models.alert_rule import AlertRuleTriggerAction
+from sentry.incidents.models.incident import IncidentStatus
+from sentry.integrations.fake_log_integration.log_provider import FakeIntegrationClient
+from sentry.integrations.services.integration import integration_service
+
+
+@AlertRuleTriggerAction.register_type(
+    "fake_log",
+    AlertRuleTriggerAction.Type.FAKE_LOG,
+    [AlertRuleTriggerAction.TargetType.SPECIFIC],
+)
+class FakeLogActionHandler(DefaultActionHandler):
```

Comment:
are you thinking we'd still register these to the legacy registries? is the `DefaultActionHandler` a legacy concept? should we be porting over a legacy base class or defining a new one?

Reply:
Not at all. I think we need to piecemeal push everything to a new unified data model and a single registry. If we can retrofit an existing registry, great, but it's not looking super likely atm.